docs/visual-basic/reference/command-line-compiler/langversion.md

```diff
@@ -18,7 +18,11 @@ Causes the compiler to accept only syntax that is included in the specified Visu
   
 ## Arguments  
  `version`  
- Required. The language version to be used during the compilation. Accepted values are `9`, `9.0`, `10`, and `10.0`.  
+ Required. The language version to be used during the compilation. Accepted values are `9`, `10`, `11`, `12`, `14`, `15`, `15.3`, `15.5`, `default` and `latest`.
+
+ Any of the whole numbers may also be specified using `.0` as the minor version, for example, `11.0`.
+
+ You can see the list of all possible values by specifying `-langversion:?` on the command line.  
   
```

Comment:
For both the C# and VB topics, the syntax to display a list of language versions (`-langversion:?`) should be mentioned.